torchvision/models/detection/roi_heads.py

```diff
@@ -322,22 +321,26 @@ def keypointrcnn_inference(x, boxes):
     return kp_probs, kp_scores
 
 
+@torch.jit.script
 def _onnx_expand_boxes(boxes, scale):
     # type: (Tensor, float) -> Tensor
-    w_half = (boxes[:, 2] - boxes[:, 0]) * .5
-    h_half = (boxes[:, 3] - boxes[:, 1]) * .5
-    x_c = (boxes[:, 2] + boxes[:, 0]) * .5
-    y_c = (boxes[:, 3] + boxes[:, 1]) * .5
-
-    w_half = w_half.to(dtype=torch.float32) * scale
-    h_half = h_half.to(dtype=torch.float32) * scale
-
-    boxes_exp0 = x_c - w_half
-    boxes_exp1 = y_c - h_half
-    boxes_exp2 = x_c + w_half
-    boxes_exp3 = y_c + h_half
-    boxes_exp = torch.stack((boxes_exp0, boxes_exp1, boxes_exp2, boxes_exp3), 1)
-    return boxes_exp
+    if boxes.numel() == 0:
+        return torch.empty(0, dtype=boxes.dtype)
```

Comment:
This will return a box with a different number of dimensions in this case, as we normally return boxes with `Nx4` dimensions. It might bring issues in the future

Reply:
Actually looks like this branch is not needed. I have cleaned this up.